jetcd-core/src/test/java/io/etcd/jetcd/SslTest.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2017 The jetcd authors
- *
+ * <p>
```

Comment:
no need to modify anything in the copy. it

Reply:
Updated